src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -288,7 +334,9 @@ public ResponseEntity<Exam> updateExamWorkingTime(@PathVariable Long courseId, @
                 int adjustedWorkingTime = Math.max(newNormalWorkingTime + timeAdjustment, 0);
                 studentExam.setWorkingTime(adjustedWorkingTime);
             }
+            // TODO: probably batch these updates?
```

Comment:
Is this meant for a follow up?

Reply:
Yes, and to start a discussion if we usually do this in artemis